frontend/app-config.yaml

```diff
@@ -1,6 +1,7 @@
 app:
   title: Hangar App
   baseUrl: http://localhost:3000
+  useApiTokenAuth: false
```

Comment:
Does it work for you? It seems like we’d need to change the schema of the config somewhere.  
Anyway, it doesn’t pick the config when I’m trying locally :confused:

Reply:
I still have to successfully run this locally.